cli/tests/validate/policies/test_none/policies/templates/gcp_iam_allow_ban_roles_v1.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2019 Google Inc.
```

Comment:
I don't want to proliferate this repo with lots of resource-specific tests. We can stick to using always_violates or the banned_resource test for basically everything.

Reply:
The main reason I'm adding resource specific tests is to check that properties are correctly mapped.